lib/plugins/aws/package/compile/events/alb/lib/validate.js

```diff
@@ -192,10 +192,10 @@ module.exports = {
   },
 
   validateMultiValueHeadersAttribute(event, functionName) {
-    if (event.alb.multiValueHeaders && !_.isBoolean(event.alb.multiValueHeaders)) {
+    if (event.alb.multiValueHeaders === true) {
```

Comment:
We should remove this condition completely

Note also that in what you propose, we throw an error when `event.alb.multiValueHeaders` is `true`, that's definitely not desired :)

Reply:
So just simple is remove function validateMultiValueHeadersAttribute. Right?
Because we check event.alb.multiValueHeaders. Right here
```
            if (event.alb.multiValueHeaders) {
              albObj.multiValueHeaders = this.validateMultiValueHeadersAttribute(
                event,
                functionName
              );
            }
```
So I think just change to 
```
            if (event.alb.multiValueHeaders) {
              albObj.multiValueHeaders = event.alb.multiValueHeaders;
            }
```
How do you think?

Reply:
> So just simple is remove function validateMultiValueHeadersAttribute

Yes, good point, exactly!

Reply:
> > So just simple is remove function validateMultiValueHeadersAttribute
> 
> Yes, good point, exactly!

Could you check again for me?